lib/test_hook.rb

```diff
@@ -1,7 +1,17 @@
 class RubyTestHook < Mumukit::Templates::FileHook
-  mashup
+  mashup do |fields| 
+    fields[2] = <<RUBY
```

Comment:
Is it necessary to reasign the `fields[2]`? Can we simply return the new string?

Reply:
As it is now, the block takes the array of (already requested) fields. A possible alternate solution without reassigning would be:
mashup do |extra, content, test|
  [extra, 
  content,
  #wrap tests here]
end

Otherwise it would also be possible for the block to simply take the tests field but having it take the array with all fields would make it easier to edit other fields should that be necessary in the future.

Reply:
I think I prefer this solution that explictly returns an array.

Reply:
Gotcha!